lib/purge-users-using-local-file.rb

```diff
@@ -0,0 +1,69 @@
+require 'zendesk_api'
+require 'rest-client'
+require 'json'
+require 'date'
+
+require_relative 'zendesk-setup.rb'
+
+# Setup vars
+
+diag = "true"
+url = "#{ENV['ZENDESK_URL']}/deleted_users/"
+tomorrow = Date.today.next_day
+lastyear = tomorrow - 365
```

Comment:
Can we replace `lastyear` with `last_year` for consistency?

Reply:
Postponed to V2